docs/site-example/domains/ffxx-def.conf

```diff
@@ -0,0 +1,49 @@
+-- This is a minimal example domain configuration for Gluon v2017.2+
+--
+-- Take a look at the documentation located at
+-- http://gluon.readthedocs.org/ for details.
+--
+-- This configuration will not work as it. You're required to make
```

Comment:
... will not work as is... there is a typo in there.

Reply:
This is copied from the current site.conf example, if there is a typo, we should fix that in the current example and I will update this PR accordingly.

Reply:
> as is

is there correct ending to that sentence. Why not update it in this PR, it's just a typo.

Reply:
i fixed it in the current example now, so you can change it in your PR, too.